### PR TITLE
ExcludesTest: Minor improvements

### PR DESCRIPTION
**Pull request by fviernau**

### CODE REVIEW

File: model/src/test/kotlin/config/ExcludesTest.kt [hunk 1]
```diff
@@ -92,22 +92,25 @@ class ExcludesTest : WordSpec() {
             "find the correct path excludes for a path" {
                 val excludes = Excludes(paths = listOf(pathExclude1, pathExclude2, pathExclude3, pathExclude4))
 
-                excludes.findPathExcludes("") should beEmpty()
-                excludes.findPathExcludes("path1") should containExactly(pathExclude1)
-                excludes.findPathExcludes("path2") should containExactly(pathExclude2)
-                excludes.findPathExcludes("test.ext") should containExactly(pathExclude3)
-                excludes.findPathExcludes("directory/test.ext") should containExactly(pathExclude3)
-                excludes.findPathExcludes("directory/file.ext") should containExactly(pathExclude3, pathExclude4)
+                excludes.apply {
```
Comment (sschuberth) [inline model/src/test/kotlin/config/ExcludesTest.kt:95]: I believe we *mostly* use `with()` to group assertions in tests, also see [apply {} vs with()](https://stackoverflow.com/a/36625603/1127485), and I have a *slight* tendency towards `with()` as it arguably reads a bit nicer in this context, but I'm not blocking this change. Any opinions?

File: model/src/test/kotlin/config/ExcludesTest.kt [hunk 2]
```diff
@@ -94,22 +92,25 @@ class ExcludesTest : WordSpec() {
             "find the correct path excludes for a path" {
                 val excludes = Excludes(paths = listOf(pathExclude1, pathExclude2, pathExclude3, pathExclude4))
 
-                excludes.findPathExcludes("") should beEmpty()
-                excludes.findPathExcludes("path1") should containExactly(pathExclude1)
-                excludes.findPathExcludes("path2") should containExactly(pathExclude2)
-                excludes.findPathExcludes("test.ext") should containExactly(pathExclude3)
-                excludes.findPathExcludes("directory/test.ext") should containExactly(pathExclude3)
-                excludes.findPathExcludes("directory/file.ext") should containExactly(pathExclude3, pathExclude4)
+                excludes.with {
```
Comment (sschuberth) [inline model/src/test/kotlin/config/ExcludesTest.kt:95]: Should be `with(excludes) {}`.

File: model/src/test/kotlin/config/ExcludesTest.kt (hunk 1 above)
Comment (fviernau) [reply model/src/test/kotlin/config/ExcludesTest.kt]: I also find it slightly nicer and it's more consistent, so let's use it.

**Review by sschuberth — APPROVED** [review]

File: model/src/test/kotlin/config/ExcludesTest.kt [hunk 3]
```diff
@@ -92,22 +92,25 @@ class ExcludesTest : WordSpec() {
...
 
             "find the correct path excludes for a project" {
                 val excludes = Excludes(paths = listOf(pathExclude1, pathExclude2, pathExclude3, pathExclude4))
 
                 setProjects(project1, project2, project3)
-
-                excludes.findPathExcludes(project1, ortResult) should containExactly(pathExclude1)
-                excludes.findPathExcludes(project2, ortResult) should containExactly(pathExclude2)
-                excludes.findPathExcludes(project3, ortResult) should beEmpty()
+                with(excludes) {
```
Comment (mnonnenmacher) [inline model/src/test/kotlin/config/ExcludesTest.kt:109]: I'd prefer to keep the empty line above to separate arrange and assert.

Comment (fviernau) [reply model/src/test/kotlin/config/ExcludesTest.kt:110]: Done.

**Review by sschuberth — APPROVED** [review]

**Pull request merged**